### PR TITLE
Add Birder as a library to model-libraries.ts

### PR DESCRIPTION
**Pull request by hassonofer**
Allow correct download stats.

### CODE REVIEW

File: packages/tasks/src/model-libraries.ts
Comment (Vaibhavs10) [inline packages/tasks/src/model-libraries.ts:117]: Thanks, just to double confirm are these the only two model checkpoints supported by the tag: https://huggingface.co/models?other=birder

File: packages/tasks/src/model-libraries.ts
Comment (Vaibhavs10) [inline packages/tasks/src/model-libraries.ts:122]: Out of curiousity, wouldn't the `config.json` not work as well? I see that the downloads already show up for: https://huggingface.co/birder-project/davit_tiny_il-all

**Review by Vaibhavs10 — COMMENTED** [review]
Thanks for the PR @hassonofer

File: packages/tasks/src/model-libraries.ts
Comment (hassonofer) [reply packages/tasks/src/model-libraries.ts:117]: Yeah, I'm still testing before uploading more...
I have around 30 more models with support of more geographical regions.

File: packages/tasks/src/model-libraries.ts
Comment (hassonofer) [reply packages/tasks/src/model-libraries.ts:122]: In the Birder library, there are a few ways to load a model (https://gitlab.com/birder/birder/-/blob/main/docs/internal/api.md?ref_type=heads)

The load with config is not a popular one, usually only the weights are being accessed.

**Comment by hassonofer** [comment]
Thank you @Vaibhavs10

File: packages/tasks/src/model-libraries.ts
Comment (julien-c) [reply packages/tasks/src/model-libraries.ts:122]: did you see by the way @sashavor? https://gitlab.com/birder/birder

File: packages/tasks/src/model-libraries.ts
Comment (Vaibhavs10) [reply packages/tasks/src/model-libraries.ts:117]: Nice! Lovely! Let us know if you need any help.

**Review by Vaibhavs10 — APPROVED** [review]
This looks good to me to merge. Let's wait for one more approval before merging.

File: packages/tasks/src/model-libraries.ts
Comment (hassonofer) [reply packages/tasks/src/model-libraries.ts:117]: Thanks a lot :)

I'm good for now, documentation is really good.

**Pull request merged**
